CONTRIBUTING.md

```diff
@@ -50,27 +50,34 @@ outlined on that page and do not file a public issue.
 * We follow the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guide.
 * In your editor, install the [editorconfig](https://editorconfig.org/) extension
   which should ensure that you are following the same standards as us.
-* Please run black and isort before opening up your PR.
-
-```
-black .
-isort .
-flake8
-```
 * Please read the [editorconfig](.editorconfig) file to understand the exact coding style preferences.
 * Please place Python code related to models in fairscale/nn. Place Python code related to optimizers
   in fairscale/optim. Place C++ extensions in fairscale/clib.
 * Please put `__all__:List[str] = []` in new `__init__.py` files for consistent importing behavior
   and less development overhead in maintaining an importing list.
+* Please setup pre-commit before opening up your PR.
 
-## Testing
+### Pre-commit
```

Comment:
nice, thanks for this @min-xu-ai, I've been willing to write this down for a while